NET_project/Benchmarks/Seahash.cs

```diff
@@ -61,22 +66,19 @@ private ulong Compute(byte[] buffer, ulong length, ulong a, ulong b, ulong c, ul
 		}
 
 		ulong excessive = length - end;
+		byte* bufferEnd = buffer + end;
 
-		if (excessive > 0)
-		{
-			a ^= buffer[^1];
+		if (excessive > 0) {
+			a ^= Read(bufferEnd);
 
-			if (excessive > 8)
-			{
-				b ^= buffer[^1];
+			if (excessive > 8) {
+				b ^= Read(bufferEnd);
```

Comment:
Both native and managed are rereading `bufferEnd` here, when I would have expected them to be reading all the remaining bytes instead. I imagine this is just a general bug in the algorithm that doesn't impact the benchmark in particular